README.md

```diff
@@ -1,62 +1,79 @@
 
-Target Platform: Toradex Verdin Development Board.
+An minimal image for the Ci4Rail Hardware including the follwing features:
 
-## cpu01-edgefarm
+- Docker
+- Read Only Filesystem
+- RW Data Partition
 
-![Build](https://concourse.ci4rail.com/api/v1/teams/main/pipelines/cpu01-edgefarm-dev/jobs/build-cpu01-edgefarm/badge)
+Target Platforms:
 
-An image for the Ci4Rail edgefarm use case (Yocto with mender, docker and iotedge support)
+- Ci4Rail Moducop CPU01
 
-Target Platform: Ci4Rail Moducop CPU01.
+### edgefarm-image
 
-## cpu01-bringup
+![Build](https://github.com/ci4rail/yocto-images/actions/workflows/edgefarm-image.yml/badge.svg?event=pull_request)
 
-![Build](https://concourse.ci4rail.com/api/v1/teams/main/pipelines/cpu01-bringup-dev/jobs/build-cpu01-bringup/badge)
+An image for the Ci4Rail EdgeFarm Services case including the following features:
 
-An image for HW platform tests and bringup. Includes mender support, but no docker and iotedge.
-Contains many tools for HW testing
+- Docker
+- Read Only Filesystem
+- RW Data Partition
+- Mender
 
-Target Platform: Ci4Rail Moducop CPU01.
+Target Platforms:
 
+- Ci4Rail Moducop CPU01
 
-# Building
-It can be either build locally using dobi or via Concourse CI.
+### devtools-image
 
+![Build](https://github.com/ci4rail/yocto-images/actions/workflows/devtools-image.yml/badge.svg?event=pull_request)
 
-## Running developer builds locally
+An image for HW platform tests and bringup including the following features:
 
-Developer builds are executed via dobi. 
+- Read Only Filesystem
+- RW Data Partition
+- Mender
+- Tools for HW testing
 
-First, enter your your mender specific data into `yocto/config/mender.env` (using template `yocto/config/mender.env.template`)
+Target Platforms:
 
-Second, enter your specific data in `default.env` (using template `default.env.template`). You can use the default setting, but MENDER_DEVICE_ID must be adapted to the device you want to use to [deploy locally built images](#Deploy images via mender).
-```
-export MENDER_DEVICE_ID=<device id from mender portal>
+- Ci4Rail Moducop CPU01
+
+## Building
+
+Images are build automatically via github actions, but this section describes, how to build images locally using dobi.
+
+First, enter credentials for mender and/or minio `config/secret.env` (using template `config/secret.env.template`).
+
+Second, enter your specific data in `config/custom.env` (using template `config/custom.env.template`). You can use the default setting, but MENDER_DEVICE_ID must be adapted to the device you want to use to [deploy locally built images via mender](#deploy-images-via-mender) and RASPBERRY_PI_IP must be adapted to the raspberry pi ip address connected to the device you want to [deploy locally built images via minio](#deploy-images-via-minio):
+
+```bash
+MENDER_DEVICE_ID=<device id from mender portal>
+RASPBERRY_PI_IP=<ip address of raspberry pi connected to the device to flash>
```

Comment:
Don't use the computer type (RPI), better use a role name (e.g. test-computer).

Reply:
ok